src/lib/builtins.pl

```diff
@@ -26,42 +26,120 @@
                      write_canonical/1, write_canonical/2,
                      write_term/2, write_term/3, writeq/1, writeq/2]).
 
+/** Builtin predicates
+
+This library, unlike the rest, is loaded by default and it exposes the most fundamental and general
+predicates of the Prolog system under the ISO standard. Basic operators, metaprogramming, exceptions,
+internal settings and basic I/O are all here.
+*/
+
 
 % unify.
+
+%% =(?X, ?Y)
+%
+% Unify two variables. This is the most basic operation of Prolog.
```

Comment:
Suggested wording, because it is less imperative: *True if X and Y can be unified.*